src/main/windows.js

```diff
@@ -24,6 +24,8 @@ export function setupWindows() {
       vibrancy: 'ultra-dark'
     });
 
+    win.setMenuBarVisibility(false);
```

Comment:
@sallar Should this not make it hidden always?

Reply:
@perry-mitchell if this is set to `false`, the user can still bring the menu up by pressing `alt`. From the docs:

> Sets whether the menu bar should be visible. If the menu bar is auto-hide, users can still bring up the menu bar by pressing the single Alt key.

But I personally thing we should make this toggle-able by having a menu item in View menu for example lables "Auto Hide Menubar" which should only be available in windows.

Reply:
Agreed. Even having it default to shown (first install) and being able to hide it _later_ with alt would be preferable.